src/SFA.DAS.AssessorService.Application.Api.UnitTests/Services/CertificateNameCapitalisationServiceTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Application.Api.Services;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Services
+{
+    [TestFixture]
+    public class CertificateNameCapitalisationServiceTests
+    {
+        private Mock<CertificateNameCapitalisationService> _certificateNameCapitalisationService;
+
+        [SetUp]
+        public void Setup()
+        {
+            _certificateNameCapitalisationService = new Mock<CertificateNameCapitalisationService>();
+        }
+
+        [TestCase(" Test")]
```

Comment:
[TestCase(" Test", "Test")] it is better to give the specific expected results rather than rely on a method.